releasenotes/notes/go-1-17-11-e57ccc89890fe6f8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    The agent is built with go1.17.11.
```

Comment:
```suggestion
    The Agent is built with Go 1.17.11.
```

Reply:
Go versions are typically written `goX.Y.Z` (see https://go.dev/doc/devel/release#go1.17 for example).  Also, this is just a backport of an existing changelog on `main`, so I'm not sure it makes sense to edit it.  What you think @cswatt?